### DISCUSSION TITLE
Instruction tuning dataset for Danish

### DISCUSSION THREAD

**Discussion by KennethEnevoldsen** [centre-for-humanities-computing/danish-foundation-models#210] [Missing pieces for Danish NLP]
# Statement of Need
Dataset for instruction tuning are important for improving the quality of Danish LLMs 

# Current status

As far as I know, I know that @kasperjunge and the DanskGPT project are working on developing an instruction-tuned dataset for Danish NLP by translating OpenOrca. Models released under this would be problematic to use in a commercial setting (due to the potential legal implication as the dataset is generated using OpenAI).

**Comment by kasperjunge** [comment]
The repo for creating the dataset can be found here:

https://github.com/kasperjunge/skolegpt-instruct-dataset

It will likely be moved to Københavns Professionshøjskoles GitHub Organisation.
